cmd/debugVerify.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+	"crypto"
+	"fmt"
+	"io"
+
+	pb "github.com/google/go-tpm-tools/proto/attest"
+	"github.com/google/go-tpm-tools/server"
+	"github.com/google/go-tpm/tpm2"
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/proto"
+)
+
+var debugVerifyCmd = &cobra.Command{
+	Use:   "debugVerify",
```

Comment:
@jkl73 and I discussed this yesterday. Can we make this "verify" and add debug as a suboption of verify? That way we can reuse the top-level verify command for talking to the GCE attestation verifier in the future. We can also make it a flag, but it seems better suited as a subcommand since debug would be orthogonal to actually verifying.

Reply:
sure @alexmwu . I have changed it to "**debugVerify** -> **verify**" and add a **debug** subcommand .